src/lib.rs

```diff
@@ -45,8 +45,8 @@ pub fn get_anchor(address: Address) -> ZomeApiResult<Anchor> {
     hdk::utils::get_as_type(address)
 }
 
-/// Gives a list of all anchors.
-pub fn get_anchors() -> ZomeApiResult<Vec<Address>> {
+/// Gives a list of all anchor types.
+pub fn get_anchor_types() -> ZomeApiResult<Vec<Address>> {
```

Comment:
This is returning the address of the `Anchor` though?

Reply:
it returns the list of addresses of the anchor types linked to the root anchor.

Reply:
ah I see yep